apple-pie/styles/style.css

```diff
@@ -0,0 +1,67 @@
+html {
+  font-family: "Futura", sans-serif;
+}
+
+header {
+  background-image: url("/apple-pie/images/apple-pie.jpg");
+  background-repeat: no-repeat;
+  background-size: 100vw;
+  background-position: center;
+  height: 50vh;
+  display: flex;
+  justify-content: center;
+  justify-items: center;
+  align-items: center;
+  margin: 0 0 8vh 0;
+}
+
+h2 {
+  font-weight: 400;
+}
+
+#imgHeading {
+  z-index: 1;
+  color: white;
+}
+
+#headImg {
```

Comment:
Do you actually use this id?

The way you ended up solving the header image is what I at least consider the best way:)